### PR TITLE
Indonesian language update

### PR DESCRIPTION
**Pull request by Khoirun0524**
They forgot to remove the dot on the string

### CODE REVIEW

File: res/values-in/strings.xml [hunk 1]
```diff
@@ -11,17 +11,17 @@
     <string name="WEEKLY">"MINGGUAN"</string>
     <string name="DAILY">"HARIAN"</string>
     <string name="Server_">"Server:"</string>
-    <string name="Game_Mode_">"Mode Permainan:"</string>
+    <string name="Game_Mode_">"Mode Permainan"</string>
```
Comment (Jeiii20) [inline res/values-in/strings.xml:14]: Please return the colon here.

File: res/values-in/strings.xml [hunk 2]
```diff
@@ -11,17 +11,17 @@
...
     <string name="Show_Names">"Tampilkan Nama"</string>
     <string name="Show_Avatars">"Tampilkan Skin"</string>
     <string name="PLAY">"MAIN"</string>
     <string name="OPTIONS">"OPSI"</string>
 
-    <string name="Avatar_">"Skins:"</string>
-    <string name="Name_">"Nama:"</string>
+    <string name="Avatar_">"Skins"</string>
+    <string name="Name_">"Nama"</string>
```
Comment (Jeiii20) [inline res/values-in/strings.xml:21]: Please return the colons here.

**Review by Jeiii20 — CHANGES_REQUESTED** [review]

File: res/values-in/strings.xml (hunk 2 above)
Comment (Khoirun0524) [reply res/values-in/strings.xml]: this needs to be translated
Screenshot_2024-12-30-11-53-14-718_software simplicial nebulous

Comment (Khoirun0524) [reply res/values-in/strings.xml]: item PLASMA_BOOST_3X_1D_XMAS/0 not found.

item PLASMA_BOOST_3X_1D_NATAL/0 tidak ditemukan.

Comment (Jeiii20) [reply res/values-in/strings.xml]: @Khoirun0524

Item names are not supposed to be translated. If you have some untranslated text to report, please use the [issue tracker](https://github.com/simplicialsoftware/nebulous-translations/issues).

**Pull request closed without merging**